docs/src/main/tut/applicative.md

```diff
@@ -43,6 +43,6 @@ into the other context:
 Applicative functors are a generalization of Monads thus allowing to express 
 effectful computations into a pure functional way.
 
-Applicative functors are generally preferred to monads when the structure 
+Applicative functors are generally preferred to Monads when the structure 
```

Comment:
How about this?

```
Like [`Functor`](functor.md) and [`Apply`](apply.md), `Applicative`
functors also compose naturally with each other. When
you compose one `Applicative` with another, the resulting `pure`
operation will lift the passed value into one context, and the result
into the other context:
```

Note:
* ```Like [`Functor`](functor.md)``` - i.e. `code` inside the link; looks like it renders nice (at least in github markdown): [`Functor`](functor.md)
* I don't think you'd want the link to `Applicative` here? Because we're on that page already.

Before I start going through the pages, probably good to know how you think about this too:
* Plurals in code style. E.g.: ```Like [Functor](functor.md) and [Apply](apply.md), `Applicative`s also...```. I do use this normally but not sure if you like it. It renders like: `Applicative`s (i.e. not that great).
* Uppercase in headers. I assume e.g. this is ok? ```## Applicative Functors & Monads```